UI/window-basic-settings.hpp

```diff
@@ -247,9 +247,13 @@ class OBSBasicSettings : public QDialog {
 	void LoadServices(bool showAll);
 	void OnOAuthStreamKeyConnected();
 	void OnAuthConnected();
+	void FetchUserInfo();
+	void FetchUserInfoFinished();
 	QString lastService;
 	int prevLangIndex;
 	bool prevBrowserAccel;
+	QPointer<QThread> fetchInfoThread;
+	QString fetchInfoResult;
```

Comment:
Should these be guarded by `#if YOUTUBE_ENABLED`?

Reply:
If you have variables guarded by preprocessor in class declarations it can lead to errors with Qt's moc thing. I probably wouldn't advise it.